aep/general/0134/aep.md.j2

```diff
@@ -65,6 +61,8 @@ separator, `snake_case` is used.
 
 Update methods implement a common request pattern:
 
+- The request **must** contain a
+  - The name of this field **must** be the singular form of the resource's
 - The request **must** contain a field for the resource.
   - The name of this field **must** be the singular form of the resource's
     name.
```

Comment:
Typo?
```suggestion
- The request **must** contain a field for the resource.
  - The name of this field **must** be the singular form of the resource's
    name.
```

Reply:
oops thanks! yes, although I'd like to consolidate this further with the section below the proto in a follow-up PR.